Add Bullet Chart (#5515)
jpmorganchase/salt-ds: site/docs/components/highcharts-theme/examples.mdx

  <ul>
    <li>
      Legend positioning isn't consistent with Figma examples due to Highcharts API limitations - [track issue](https://github.com/jpmorganchase/salt-ds/issues/5548).
    </li>
    <li>

>       Plot bands aren't supported as part of this release due to v10 accessibility and token issues. If you need to use plot bands, you need to ensure that appropriate color contrast is maintained - [track issue](https://github.com/jpmorganchase/salt-ds/issues/5548 )

Reviewer: ```suggestion
      Plot bands aren't currently supported due to v10 accessibility and token issues. If you need to use plot bands, you need to ensure that appropriate color contrast is maintained - [track issue](https://github.com/jpmorganchase/salt-ds/issues/5548 )
```
